codalab/model/bundle_model.py

```diff
@@ -2184,31 +2164,195 @@ def get_user(self, user_id=None, username=None, check_active=True):
             user_ids = [user_id]
         if username is not None:
             usernames = [username]
-        result = self.get_users(user_ids, usernames, check_active)
-        if result:
-            return result[0]
+        result = self.get_users(user_ids=user_ids, usernames=usernames, check_active=check_active)
+        if result['results']:
+            return result['results'][0]
         return None
 
-    def get_users(self, user_ids=None, usernames=None, check_active=True):
+    def get_users(self, keywords=None, user_ids=None, usernames=None, check_active=True):
         """
-        Get users.
+        Get user stats.
 
-        :param user_ids: user ids of users to fetch
+        Returns a bundle search result dict where:
+            result: list of bundle uuids matching search criteria in order
+                        specified for bundle searches
+                    single number value for aggregate searches(.count)
+            is_aggregate: True for aggregate searches, False otherwise
+        Each keyword is either:
```

Comment:
We probably shouldn't duplicate the documentation, since this is already documented in the `uls` CLI command and it may be hard to keep both docstrings in sync. Can you maybe replace these details about keywords to something like "see the documentation for cl uls for information about keyword structure"?